src/lib/v7/v7.coffee

```diff
@@ -0,0 +1,67 @@
+SportApi = require '../sport-api'
+
+class V7 extends SportApi
+  version: 7
+
+  getWeeklySchedule: (params, callback) ->
+    params = this.getYearSeasonWeekParams(params)
+    this.getResource '/%(year)s/%(season)s/%(week)s/schedule.xml', params, callback
+
+  getExtendedBoxscore: (params, callback) ->
+    this.getResource '/games/%(gameId)s/extended-boxscore.xml', params, callback
```

Comment:
Per the most recent data this will need to be updated from `extended-boxscore.xml` to `boxscore.xml`